<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealEstateBase</class>
 <widget class="QWidget" name="RealEstateBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView_estate">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>531</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Выберите</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_district">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>59</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Район</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_district">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Выберите</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_square">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Площадь, м2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_floor_amount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Выберите</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_price">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>372</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена, USD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_price_from">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>400</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>От</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_price_to">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>До</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Карточка объекта</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>81</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_floor_amount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Кол-во комнат</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_year">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>45</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Год</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>81</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_year_to">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>До</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_year_from">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>От</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_square_to">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>До</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_square_from">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>От</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
